js.js

```diff
@@ -0,0 +1,24 @@
+function myFunction_About()

```

Comment:
Please look at my branch. I updated your code. Then  you can merge my code into your branch. Just click merge pull request.